src/main/java/com/fasterxml/jackson/core/io/BigDecimalParser.java

```diff
@@ -60,6 +64,40 @@ public static BigDecimal parse(char[] chars) {
         return parse(chars, 0, chars.length);
     }
 
+    public static BigDecimal parseWithFastParser(final String valueStr) {
```

Comment:
Maybe also rename this as `BigNumberParser`?

Oh, this was added in 2.13 and is public.

Ok... Let's actually add separate `BigIntegerParser` for clarity. May seem silly but I think it just makes sense here to separate.

Reply:
done